pkg/kubectl/cmd/expose.go

```diff
@@ -70,7 +70,7 @@ func NewCmdExposeService(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	cmd.Flags().Bool("dry-run", false, "If true, only print the object that would be sent, without creating it.")
 	cmd.Flags().String("container-port", "", "Synonym for --target-port")
 	cmd.Flags().String("target-port", "", "Name or number for the port on the container that the service should direct traffic to. Optional.")
-	cmd.Flags().String("public-ip", "", "Name of a public IP address to set for the service. The service will be assigned this IP in addition to its generated service IP.")
+	cmd.Flags().String("external-ips", "", "External IP addresses(comma-separated) to set for the service. The service can be accessed by these IPs in addition to its generated service IP.")
```

Comment:
@eparis it doesn't look like pflags.StringSlice and Cobra are well integrated - interesting to you?